funfact/lang/_tsrex.py

```diff
@@ -230,7 +230,7 @@ def __truediv__(self, rhs):
 
     @as_tsrex
     def __pow__(self, rhs):
-        return _pow(_as_node(self), _as_node(rhs))
+        return _binary(_as_node(self), _as_node(rhs), 3, 'float_power')
```

Comment:
`float_power` seems a bit of a weird name to me. maybe just `power` is better? Similar to the name in NumPy.

Reply:
Does the backend have to implement a `float_power` method for this to work?

Reply:
That's what I found out, `float_power` is the method that gets implemented in all three backends. Other than always promoting integers to floating points (which we almost always want), there is no side effect. :grin: